README.md

```diff
@@ -0,0 +1,50 @@
+Automation wrapper for burst_io
+
+Description:
+            This wrapper runs a program that performs burst io activity.
+	    The program will perform a specified IO for a designated amount
+	    of time and then sleeps for another amount of time.  Once the program
+            wakes up, it will terminate if the overall time of the run has expired,
+            else it will perform another IO iteration.
+	    and does not deal with numa.
+  
+Location of underlying workload: part of the github kit
+
+Packages required: gcc,bc,wget
+
+To run:
+[root@hawkeye ~]# git clone https://github.com/redhat-performance/io_burst-wrapper
+[root@hawkeye ~]# io_burst-wrapper/io_burst/burst_io.sh
```

Comment:
Surround with \`\`\`'s

Reply:
You forgot to close the \`\`\` at line 20.

Line 17 doesn't need to be included in the \`\`\`

Reply:
17 does not matter if it is or not included, did so the entire block is enclosed.
The ending ``` are at the file.  All I did was move the opening ```  to be before the sample usage, instead of before the usage of the program.